enterprise/backup/src/main/java/org/neo4j/backup/BackupTool.java

```diff
@@ -69,9 +68,9 @@
     private static final String VERIFY = "verify";
     private static final String CONFIG = "config";
 
-
     private static final String TIMEOUT = "timeout";
     private static final String FORENSICS = "gather-forensics";
+    private static final String FULL_ID_GENERATOR_REBUILD = "full-id-generator-rebuild";
```

Comment:
The logic around the default value for this parameter is confusing.

The `full-id-generator-rebuild` flag defaults to false. But is then negated when assigned to the `rebuild_idgenerators_fast` config option, meaning that this option will default to true. This means that by default the id generator files will be removed.

This is probably the right thing to do, but the names of these flags and the actual behaviour does not match perfectly, so the flow gets confusing.

I'm guessing it also reuses the (prior existing) implementation for rebuilding id generators. I think we at least need a lot more documentation around this. But preferably the code should be refactored to make the effect of the flags, and the behaviour more apparent.